core/domain/exp_jobs_one_off_test.py

```diff
@@ -763,6 +764,114 @@ def test_that_mapreduce_job_works(self):
+                'testing</i></b>in <b>progress</b><p>for migration</p>'
+            )
+        }
+        content2_dict = {
+            'content_id': 'content',
+            'html': (
+                'Here is test case <a href="https://github.com">'
+                '<oppia-noninteractive-image filepath-with-value="'
+                '&amp;quot;random1.png&amp;quot;" caption-with-value="&amp;'
+                'quot;&amp;quot;" alt-with-value="&amp;'
+                'quot;&amp;quot;"></oppia-noninteractive-image><p>'
+                ' testing in progress</p>'
+            )
+        }
+        state1.update_content(content1_dict)
+        state2.update_content(content2_dict)
+
+        default_outcome_dict1 = {
+            'dest': 'State2',
+            'feedback': {
+                'content_id': 'default_outcome',
+                'html': (
+                    '<p>Sorry, it doesn\'t look like your <span>program '
+                    '</span>prints output</p>.<blockquote><p> Could you get '
+                    'it to print something?</p></blockquote> Can do this by '
+                    'using statement like prints. <br> You can ask any if you '
+                    'have<oppia-noninteractive-link url-with-value="&amp;quot;'
+                    'https://www.example.com&amp;quot;" text-with-value="'
+                    '&amp;quot;Here&amp;quot;"></oppia-noninteractive-link>.'
+                )
+            },
+            'labelled_as_correct': False,
+            'param_changes': [],
+            'refresher_exploration_id': None,
+            'missing_prerequisite_skill_id': None
+        }
+        default_outcome_dict2 = {
+            'dest': 'State1',
+            'feedback': {
+                'content_id': 'default_outcome',
+                'html': (
+                    '<ol><li>This is last case</li><oppia-noninteractive-image '
+                    'filepath-with-value="&amp;quot;random2.png&amp;quot;" '
+                    'caption-with-value="&amp;quot;&amp;quot;" alt-with-value='
+                    '"&amp;quot;&amp;quot;">'
+                    '</oppia-noninteractive-image></ol>'
+                )
+            },
+            'labelled_as_correct': False,
+            'param_changes': [],
+            'refresher_exploration_id': None,
+            'missing_prerequisite_skill_id': None
+        }
+
+        state1.update_interaction_default_outcome(default_outcome_dict1)
+        state2.update_interaction_default_outcome(default_outcome_dict2)
+        exp_services.save_new_exploration(self.USER, exploration)
+
+        job_id = exp_jobs_one_off.CreateVersionsOfImageJob.create_new()
+        exp_jobs_one_off.CreateVersionsOfImageJob.enqueue(
+            job_id)
+        self.process_and_flush_pending_tasks()
+
+        actual_output = (
+            exp_jobs_one_off.CreateVersionsOfImageJob.get_output(
+                job_id))
+
+        expected_output = [
```

Comment:
You should also check the contents of the file system, before and after.

Reply:
Done